app/lib/actions/katello/system/update.rb

```diff
@@ -17,6 +17,8 @@ def plan(system, sys_params)
           end
 
           if system.foreman_host.subscription_facet
+            release_version = system.releaseVer.is_a?(Hash) ? system.releaseVer['releaseVer'] : system.releaseVer
+            system.foreman_host.subscription_facet.release_version = release_version
```

Comment:
I agree this i really ugly, but the System object can present releaseVer as a string or as a Hash, really poorly designed and will go away when systems go away.